vars/capsulePipelineWrapper.groovy

```diff
@@ -229,11 +219,11 @@ def call() {
             stage('Restart Network') {
                 when {
                     expression {
-                        params.ACTION == 'RESTART' && params.UNSAFE_RESET_ALL
+                        params.ACTION == 'RESTART' && params.UNSAFE_RESET_ALL // TODO: This probably should be applied to START as well
```

Comment:
about the comment: I would say that we should also allow a clean start, without `unsafe-reset-all`.

Reply:
Yes yes currently we have:

- Start 
- Restart
- Restart + Unsafe reset all
- Stop :)